api/lib/block-listeners/base-node-subscription.js

```diff
@@ -58,6 +64,15 @@ class BaseNodeSubscription {
     )
   }
 
+  async pollForValidatorsProfiles(validators) {
+    const dataSource = this.getDataSource()
+    this.getValidatorProfiles(validators, dataSource)
+
+    this.validatorProfilePollingTimeout = setTimeout(async () => {
```

Comment:
```suggestion
    setTimeout(async () => {
```

Reply:
polling for proposals has its own :eyes:

Reply:
do you ever use `validatorProfilePollingTimeout`?

Reply:
ah no. I see